cmd/db.go

```diff
@@ -183,10 +183,13 @@ var (
 		},
 	}
 
+	noSeed bool
+
 	dbResetCmd = &cobra.Command{
 		Use:   "reset",
 		Short: "Resets the local database to current migrations",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			utils.Config.Db.Seed.Enabled = !noSeed
```

Comment:
```suggestion
			if noSeed {
				utils.Config.Db.Seed.Enabled = false
			}
```

If config is disabled, but flag is left at default, we still want to respect the config value.